### DISCUSSION TITLE
Open Shell in Microsoft Store

### DISCUSSION THREAD

**Discussion by ananyapraharsh** [Open-Shell/Open-Shell-Menu#1064] [General]
I'm a PM at Microsoft, working in the team responsible for the Microsoft Store on Windows. We would love to have Open Shell as an app in the Store. We have made significant changes to our Store in the last year to reduce friction for our partners, including the ability to let you publish a traditional desktop win32 application. Full details can be found here: https://developer.microsoft.com/en-us/microsoft-store/. 

If there is someone who is working on Open Shell and would like to have a discussion on having Open Shell in the Microsoft Store, please let me know. It would be great to talk to you!

**Comment by blackcrack** [comment]
Hi @ananyapraharsh  Ananya, 

well, pretty nice for know this app to be appreciated, but before , should you give programmer to make this Open-Shell  Win11 Compatible ..
to have in the future a Open-Shell who well works in WinNT also in future.
I guess, OpenShell has an urgent right to exist for make the possible for the users for have his own Startmenu like the Users it want and make it possible for use Skins ...
So would it grate to have a MS-Group where Support this Software and programming it further for the Peoples who love it.
And it is a  relative huge User community who use this Open Source Software because, they can choose which Menu they are use.
Therefore should it supported by MS programming employees to hold this useful Software..
Therewith do i ask for help, please help Open-Shell .

best
Blacky
